src/Jackett.Common/Indexers/HDTorrents.cs

```diff
@@ -144,21 +142,29 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 };
                 var hasFreeleech = freeleechRanks.Contains(rank);
 
-                var rows = dom.QuerySelectorAll(".mainblockcontenttt > tbody > tr:has(a[href^=\"details.php?id=\"])");
+                var rows = dom.QuerySelectorAll("table.mainblockcontenttt tr:has(td.mainblockcontent)").Skip(1);
                 foreach (var row in rows)
                 {
                     var release = new ReleaseInfo();
 
-                    release.Title = row.QuerySelector("td.mainblockcontent b a").TextContent;
-                    release.Description = row.QuerySelector("td:nth-child(3) > span").TextContent;
+                    var mainLink = row.Children[2].QuerySelector("a");
+
+                    release.Title = row.Children[2].QuerySelector("a").TextContent;
```

Comment:
mainLink.TextContent

Reply:
This one is kinda embarrassing, since it's the whole reason I separated it into a variable and I missed it.